code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -227,7 +227,8 @@
 	if(holder && holder.my_atom)
 		var/turf/open/T = get_turf(holder.my_atom)
 		if(istype(T))
-			T.atmos_spawn_air("nitrogen=50;TEMP=2.7")
+			var/datum/gas/gastype = /datum/gas/nitrogen
+			T.atmos_spawn_air("[initial(gastype.id)]=50;TEMP=2.7")
```

Comment:
would probably be good to update this to not use atmos_spawn_air

Reply:
Any suggestions? That code is what @ninjanomnom suggested as a replacement.

Reply:
Actually I'll have to replace this more generally in a different PR, ignore me